windows/src/desktop/kmbrowserhost/kmbrowserhost.dproj

```diff
@@ -167,6 +168,20 @@
                         <RemoteDir>classes</RemoteDir>
                         <Operation>1</Operation>
                     </Platform>
+                    <Platform Name="Android64">
+                        <RemoteDir>classes</RemoteDir>
+                        <Operation>1</Operation>
+                    </Platform>
+                </DeployClass>
+                <DeployClass Name="AndroidFileProvider">
+                    <Platform Name="Android">
+                        <RemoteDir>res\xml</RemoteDir>
+                        <Operation>1</Operation>
```

Comment:
A see that the bulk of the `.dproj` changes are a lot of new entries like this.  Intentional?

Reply:
This is a side-effect of loading the .dproj in latest version of Delphi. It adds a lot of cross-platform guff. Nothing that we need to worry about.